rest_sensors.cpp

```diff
@@ -963,6 +958,15 @@ int DeRestPluginPrivate::updateSensor(const ApiRequest &req, ApiResponse &rsp)
         }
     }
 
+    if (map.contains("config")) // optional
```

Comment:
Not sure if I like this, since other parts of the API don't do this kind of deep-copy and only update respective attributes on the nesting level given by the URL. Does the Hue bridge support this?

Reply:
Yep, the Hue app sends a {"config": {"usertest": true}} when setting/testing the sensitivity of a Hue motion sensor... I agree it's a bit inconsistent, but it does seem to be supported.

Reply:
Haven't seen that one before (and, afaik, it's undocumented).  But the Hue API uses nested objects in the body on multiple occasions.  E.g. to reset the HomeKit server, PUT `/config` with a body of `{"homekit": {"factoryreset": true}}`.

Reply:
Yes, it seems undocumented. As in, usertest is described as an attribute of the config object, but the docs way to change a config is indeed to have the url end in .../config. I only found this by chance when adding the motion sensor.

Reply:
Interesting, since the Hue bridge uses this we can add it too. Albeit I would prefer supporting just the documented ways or find a generic solution for nested requests (might be difficult).

Reply:
API v2...